src/joinVoiceChannel.ts

```diff
@@ -1,11 +1,18 @@
-import { VoiceChannel } from 'discord.js';
 import { createVoiceConnection } from './VoiceConnection';
 import { JoinConfig } from './DataStore';
 
 /**
  * The options that can be given when joining a voice channel
  */
 export interface JoinVoiceChannelOptions {
+	/**
+	 * The ID of the voice channel to join
+	 */
+	channelId: string;
```

Comment:
Snowflake type

Reply:
Resolved in https://github.com/discordjs/voice/pull/65/commits/54f4d50d344cb7353147c09e1b26eef7bdcc28fd